rules/false_positives/wolfictl.yara

```diff
@@ -0,0 +1,14 @@
+rule wolfictl_binary : override {
+  meta:
+    curl_tor_chmod_relative_run = "high"
+    description = "wolfictl"
+    original_severity = "critical"
+  strings:
+    $chainguard = "chainguard.dev"
+    $false_positive_string = "domaingophertelnetreturn.locallisten.onionndots:sendtoip"
```

Comment:
add `listen.onionndots` as a `not_go` anti-match string within the `curl_tor_chmod_relative_run` rule. The rule wasn't intended to capture it.

Reply:
Updated in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).